wordpress-mobile/WordPress-Android
# Issue/hide edit action for video in custom picker

This PR hides "Edit" action for video from the bottom bar added in https://github.com/wordpress-mobile/WordPress-Android/pull/11646 in the custom media picker.

### To test 

- Create a new post in gutenberg
- Add a video block
- Select "Choose from device"
- Select a video
- Notice that "checkmark" done action in the toolbar is visible and bottom bar with insert/ edit actions is not shown.

PR submission checklist:
